src/components/AcceptedValuesField/AcceptedValuesField.js

```diff
@@ -39,7 +40,7 @@ export const AcceptedValuesField = ({
     data.forEach(item => {
       acceptedValues = {
         ...acceptedValues,
-        [item.id]: item.name,
+        [item.id]: getAcceptedValuesOption(item, 'name', 'code'),
```

Comment:
Consider retrieving `optionField` and `fieldToAdd` values from the props because it will not be working if you will need to concatenate 'name' and some other field

Reply:
It's not a value, it's name of field, from which value should be taken

Reply:
I see, I mean that field names can be different, e.g. for one field you need to concatenate 'name' and 'code', for another one you may need to concatenate 'name' and 'someFieldName'

Reply:
Consider passing something like the template string and parse it in this component. For example:
`optionTemplate: '${name} (${code})'` 
We have something similar to it in the Control component, where we replace `**ri**` with `repeatableIndex`